src/toil/test/src/regularLogTest.py

```diff
@@ -0,0 +1,36 @@
+# Copyright (C) 2015 UCSC Computational Genomics Lab
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import subprocess
+import sys
+from toil.test import ToilTest
+from toil.test.mesos import helloWorld
+
+
+class RegularLogTest(ToilTest):
+
+    def testLogToMaster(self):
+        toilOutput = subprocess.check_output([sys.executable,
+                                              '-m', helloWorld.__name__,
+                                              './toilTest',
+                                              '--clean=always',
+                                              '--logLevel=info'], stderr=subprocess.STDOUT)
+        assert "the child job is now running!" in toilOutput
```

Comment:
… and use that variable here. That makes the connection clear. Without it, an unsuspecting user might change the string in helloWorld.py without changing it here.